app/uk/gov/hmrc/exports/services/WCOPointerMappingService.scala

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmrc.exports.services
+
+import javax.inject.Inject
+import uk.gov.hmrc.exports.models.Pointer
+
+class WCOPointerMappingService @Inject()(){
```

Comment:
So this is the entry point to the Pointer Mapping

If you can construct a WCO `Pointer` from the Notification XML, pass it to this method and it will convert it to the exports equivalent.

I've added a method that accepts a `Seq[Pointer]` if it's useful (depends if you want to batch up the mapping or not), we can delete it if not.